test/e2e/nsx_security_policy_test.go

```diff
@@ -322,3 +313,24 @@ func assureSecurityPolicyReady(t *testing.T, ns, spName string) {
 	})
 	require.NoError(t, err)
 }
+
+func assureTraffic(ns, podname, containername, ip string, port int32, interval, timeout time.Duration) error {
```

Comment:
It looks most code is copied from precreated_vpc_test.go? If you expect to share them in multiple files, better to place such logic into a function in framework.go, and call the function in each test file?

Reply:
ok